### ISSUE TITLE
Error on webhook update

### ISSUE THREAD

**Issue by vytska** [SparkPost/node-sparkpost#164]
When updating webhook I get:

```
{"cause":{"name":"SparkPostError","errors":[{"param":"id","value":"4c859a40-6b87-11e6-b486-192bb1c1783d","message":"Unknown parameter"}],"statusCode":400},"isOperational":true,"errors":[{"param":"id","value":"4c859a40-6b87-11e6-b486-192bb1c1783d","message":"Unknown parameter"}],"statusCode":400}
```

This is probably because ID is sent with update object.

**Comment by aydrian** [comment]
It appears you are correct. `id` should have been removed from the `webhook` object before it was passed into the `json` param.
We should also be checking that `webhook.id` exists.

**Comment by colestrode** [comment]
Our API should ignore additional parameters, this may be a bug in the API. But yeah, we should at least update the client lib to handle this.

**Comment by colestrode** [comment]
I confirmed this is an issue in our API and have filed an internal bug for it. Thanks for reporting @vytska.

**Comment by aydrian** [comment]
This has been fixed and published as v1.3.8
